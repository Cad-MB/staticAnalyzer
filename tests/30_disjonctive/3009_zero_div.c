{
    int x;
    int y;
    x = 10;
    y = 0;
    if (y != 0) {
      x = x / y;
    } else {
      x = 42;
    }
    print_all;
  }