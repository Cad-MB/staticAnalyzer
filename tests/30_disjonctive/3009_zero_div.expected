tests/30_disjonctive/3009_zero_div.c:11.4-14: [ x in {[42;42]}, y in {[0;0]} ]
analysis ended